mne/io/ctf/eeg.py

```diff
@@ -49,3 +53,46 @@ def _read_eeg(directory):
                     eeg['np'] += 1
     logger.info('    Separate EEG position data file read.')
     return eeg
+
+
+def _read_pos(directory, transformations):
+    """Read the .pos file and return eeg positions as digitizer extra points.
+    """
+    fname = [join(directory, f) for f in listdir(directory) if
+             f.endswith('.pos')]
+    if len(fname) < 1:
+        return list()
+    elif len(fname) > 1:
+        warnings.warn('    Found multiple pos files. Extra digitizer points '
+                      'not added.')
+        return list()
+    logger.info('    Reading digitizer points from %s...' % fname)
+    if transformations['t_ctf_head_head'] is None:
+        warnings.warn('    No transformation found. Extra digitizer ponts not '
+                      'added')
+        return list()
+    fname = fname[0]
+    digs = list()
+    i = 1000
```

Comment:
Here I made so that the eeg positions go from 1000 upwards if the id field is not an int. Maybe this should also be the case even when there are ints at the beginning. I actually made a kind of a mistake of adding to the testing dataset a pos file that has overlapping ids with existing dig points.